feature.py

```diff
@@ -0,0 +1,13 @@
+import calendar
+
+
+def is_leap_year_v1(year):
+    return (year % 4 == 0 and year % 100 != 0) or year % 400 == 0
+
+
+def find_leap_year_v2(year):
+    result = False
+    if calendar.isleap(year):
```

Comment:
This block can be simplified. as you checking isLeap for boolean value there is no need to write down such construction. In other words you can replace it be return calendar.isleap(year). As find_leap_year_v2 contains only one lineof code this function can be deprecated.